include/oneapi/dpl/internal/async_impl/async_impl_hetero.h

```diff
@@ -48,7 +48,7 @@ __pattern_walk1_async(__hetero_tag<_BackendTag>, _ExecutionPolicy&& __exec, _For
     return __future_obj;
 }
 
-template <typename _IsSync = ::std::false_type,
+template <typename _WaitMode = __par_backend_hetero::__async_mode,
```

Comment:
Do we really need this tag for `__pattern_walk2_async`? I mean, it is supposed to be asynchronous, is not it? And of course currently it is not used with other than the default template argument.

Reply:
The source idea was to have the same format for `__pattern_walk2_async` and for `__pattern_walk2`

Reply:
I know; and I know it is not in your patch. But my question is, is it really needed? And we are changing the pattern slightly anyway, so I think it's appropriate to revisit this.

Reply:
From current use cases state it's really not required.

Reply:
Fixed: removed.